.devcontainer/devcontainer.json

```diff
@@ -10,6 +10,7 @@
         "--network",
         "host"
     ],
+    "postCreateCommand": "make install && pip install -e .",
```

Comment:
>make install && pip install -e . [](http://example.com/codeflow?start=26&length=32)

I don't know the difference between these 2 commands but I guess we need only one at the time?